apps/agent/app/src/components/AgentHelp.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { GU, Info, Link, useTheme } from '@aragon/ui'
+import AgentSvg from './AgentSvg'
+
+function AgentHelp() {
+  const theme = useTheme()
+
+  return (
+    <React.Fragment>
+      <Info
+        css={`
+          margin-bottom: ${2 * GU}px;
+        `}
+      >
+        <div
+          css={`
+            display: grid;
+            grid-template-rows: auto auto;
+            grid-template-columns: auto 1fr;
+            grid-column-gap: ${2 * GU}px;
+            align-items: center;
+          `}
+        >
+          <div
+            css={`
+              width: 46px;
+              height: 46px;
+            `}
+          >
+            <AgentSvg />
+          </div>
+          <div>
+            <div
+              css={`
+                margin-bottom: ${0.5 * GU}px;
+              `}
+            >
+              The Agent app can be used to interact with external contracts.
+            </div>
+            <div>
+              <Link
+                css={`
+                  color: ${theme.indicatorContent};
+                `}
```

Comment:
This color doesn't exist in aragonUI, so this is why it shows up as pink in the example. :) You probably meant [`tagIndicatorContent`](https://ui.aragon.org/colors/). That said though, why are we switching colors?

Reply:
To be completely honest, I just copied what appeared in the [Agent modal](https://github.com/aragon/aragon-apps/blob/master/apps/agent/app/src/components/FrameModal.js#L37), so I have no reason for switching colors other than that lol

Reply:
Ah haha I see! That component was made long ago, and the theme colors have changed a lot. :) Let's revert the color change and leave the standard `Link` color, and we can also safely remove this modal, as the "Learn More" will now be a external link, rendering it unused.